prow/crier/crier.go

```diff
@@ -129,10 +129,10 @@ func parseIssueComments(r Report, ics []github.IssueComment) ([]int, []string) {
 		}
 		previousComments = append(previousComments, ic.ID)
 		var tracking bool
-		for _, line := range strings.Split(ic.Body, "\r\n") {
+		for _, line := range strings.Split(ic.Body, "\n") {
 			if strings.HasPrefix(line, "---") {
 				tracking = true
-			} else if len(line) == 0 {
+			} else if len(strings.TrimSpace(line)) == 0 {
```

Comment:
I think you should move this up (line = strings.TrimSpace(line)), since otherwise you'll add things to entries that have a \r in them.